roles/pvc/mysql_pvc/files/mysql-persistent-template.yml

```diff
@@ -69,8 +69,6 @@ spec:
       labels:
         name: mysql
         app: mysql
-      annotations:
-        backup.velero.io/backup-volumes: mysql-data
```

Comment:
This line should either be deleted, or changed to point to the current PVC name.